.github/workflows/statuspage.yml

```diff
@@ -0,0 +1,27 @@
+ name: Rapid7 Status
+
+ on: [release]
```

Comment:
@mrinehart-r7 Can you confirm we want this workflow to run specifically on the release event? https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release